website/content/sdk/guides/intros/stocks/stocks-screener.md

```diff
@@ -0,0 +1,314 @@
+
+One way to use the screener is to feed it the least amount of variables possible. In this next example, the preset file contains only one filter; requesting data for all constituents of the S&P 500 Index.
+
+```console
+# Author of preset: Danglewood
+# Description: S&P Index
+
+[General]
+Order = Market Cap.
+
+[Descriptive]
+Index = S&P 500
+
+[Fundamental]
+
+[Technical]
+```
+
+Copy and past the block above into any text editor; then save the file, as `sp500_filter.ini`, to the OpenBBUserData folder.
+
+```python
+sp500_overview = openbb.stocks.screener.screener_data(preset_loaded='sp500_filter.ini', data_type = 'overview', limit = 500)
+sp500_overview = sp500_overview.convert_dtypes()
+
+sp500_overview
+```
+
+|    | Ticker   | Company               | Sector                 | Industry                       | Country   |   Market Cap |   P/E |   Price |   Change |   Volume |
+|---:|:---------|:----------------------|:-----------------------|:-------------------------------|:----------|-------------:|------:|--------:|---------:|---------:|
+|  0 | AAPL     | Apple Inc.            | Technology             | Consumer Electronics           | USA       |  2.23252e+12 | 23.14 |  141.17 |  -0.0211 | 83578852 |
+|  1 | MSFT     | Microsoft Corporation | Technology             | Software - Infrastructure      | USA       |  1.77308e+12 | 25.9  |  240.33 |  -0.0059 | 17929705 |
+|  2 | GOOG     | Alphabet Inc.         | Communication Services | Internet Content & Information | USA       |  1.23434e+12 | 18.68 |   95.44 |  -0.0084 | 20145081 |
+|  3 | GOOGL    | Alphabet Inc.         | Communication Services | Internet Content & Information | USA       |  1.21605e+12 | 19.51 |   95.19 |  -0.009  | 20047848 |
+|  4 | AMZN     | Amazon.com, Inc.      | Consumer Cyclical      | Internet Retail                | USA       |  9.3352e+11  | 85.26 |   92.42 |  -0.0163 | 65245723 |
+...continued...
+| 498 | LUMN     | Lumen Technologies, Inc. | Communication Services | Telecom Services              | USA       |     5.61e+09 |  2.74 |    5.47 |  -0.0091 | 31885469 |
+| 499 | PENN     | PENN Entertainment, Inc. | Consumer Cyclical      | Resorts & Casinos             | USA       |     5.35e+09 | 24.72 |   34.07 |   0.0119 |  1689121 |
+| 500 | NWL      | Newell Brands Inc.       | Consumer Defensive     | Household & Personal Products | USA       |     5.27e+09 |  9.6  |   12.87 |  -0.0191 |  5725896 |
+| 501 | VNO      | Vornado Realty Trust     | Real Estate            | REIT - Office                 | USA       |     4.86e+09 | 50.72 |   25.31 |   0.0218 |  2560781 |
+| 502 | PVH      | PVH Corp.                | Consumer Cyclical      | Apparel Manufacturing         | USA       |     4.21e+09 |  4.96 |   64.88 |   0.0399 |  1100532 |
+
+With this preset, the five different data types can be quickly assembled as one dataframe. Occasionally, an error may pop up when the screener is run; but, it should remedy itself if the command is repeated.
```

Comment:
Please remove all info further from here. We don't have to inform the user that it is possible you receive an error. Next to that the sample code is too large and you already explain much of this above. We do not have to include this. The output also doesn't look as nice as the tables.

Reply:
Done. Cut it out and cut straight to the chase with the sample script.